src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -546,6 +550,18 @@ public function resolveConnection($connection)
         return $this->resolver->connection($connection ?: $this->connection);
     }
 
+    /**
+     * Restore connection to default instance from .env file.
+     *
+     * @return void
+     */
+    protected function restoreDefaultConnection()
+    {
+        if ($defaultConnection = env('DB_CONNECTION')) {
```

Comment:
You are assigning instead of comparing:
```suggestion
        if ($defaultConnection === env('DB_CONNECTION')) {
```

Reply:
Yes, I want to assign in this case as that variable is then used in the assignment below. This `if` is to check if it's `null`. Does that make sense?